plugin/src/main/java/org/gradle/testretry/internal/visitors/SpockParameterClassVisitor.java

```diff
@@ -36,6 +37,11 @@
 public class SpockParameterClassVisitor extends ClassVisitor {
 
     private static final Set<Character> REGEX_CHARS = setOf(new char[]{'<', '(', '[', '{', '\\', '^', '-', '=', '$', '!', '|', ']', '}', ')', '?', '*', '+', '.', '>'});
+    //A valid Java identifier https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-3.8 including methods
+    public static final String SPOCK_PARAM_PATTERN = "#[\\p{L}\\d$_.()&&[^#\\s]]+";
+    public static final String PARAM_PLACEHOLDER = "#param";
+    public static final String WILDCARD_SUFFIX = ".*";
+    public static final String WILDCARD = WILDCARD_SUFFIX;
```

Comment:
Why is that constant required, as it's the same value as `WILDCARD_SUFFIX`?

Reply:
just for better readability

Reply:
I checked twice when reading the regex replacement ;). But lets keep it as it is.